wsrep_api.h

```diff
@@ -246,6 +258,19 @@ extern int
 wsrep_gtid_print(const wsrep_gtid_t* gtid, char* str, size_t str_len);
 
 /*!
+ * @brief Compare two GTIDs
+ *
+ * First the UUID components are compared according to wsrep_uuid_compare().
+ * If they match, the return value is -1, 0, 1 if the SEQNO component of lhs
+ * is respectively smaller, equal, or greater than the SEQNO component of rhs.
+ *
+ * @return -1, 0, 1 if lhs is respectively smaller, equal, or greater than rhs
+ */
+extern int
+wsrep_gtid_compare (const wsrep_gtid_t* lhs, const wsrep_gtid_t* rhs);
```

Comment:
Yes, after second thought I don't see any use for ```wsrep_gtid_compare()``` and would consider it even harmful. Better to remove and make uuid/seqno part comparison more explicit in code.